src/frontend/komponenter/Fagsak/Behandle/FastsettVedtak.tsx

```diff
@@ -1,143 +1,100 @@
+import * as moment from 'moment';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
 import { Nesteknapp } from 'nav-frontend-ikonknapper';
-import { Panel } from 'nav-frontend-paneler';
-import { Input, RadioPanelGruppe, Select, SkjemaGruppe } from 'nav-frontend-skjema';
-import { Element, Normaltekst, Systemtittel, Undertittel } from 'nav-frontend-typografi';
+import { Normaltekst, Systemtittel } from 'nav-frontend-typografi';
 import * as React from 'react';
-import { IBarnBeregning } from '../../../typer/behandle';
-import { IFagsak, sakstyper } from '../../../typer/fagsak';
-import { IFelt, Valideringsstatus } from '../../../typer/felt';
-import useFastsettReducer from './useFastsettReducer';
+import { useHistory } from 'react-router';
+import { axiosRequest } from '../../../api/axios';
+import { IBehandling, IFagsak } from '../../../typer/fagsak';
+import { Valideringsstatus } from '../../../typer/felt';
+import { Ressurs, RessursStatus } from '../../../typer/ressurs';
+import {
+    actions,
+    useFastsettVedtakContext,
+    useFastsettVedtakDispatch,
+} from './FastsettVedtakProvider';
+import FastsettVedtakSkjema from './FastsettVedtakSkjema';
 
 interface IProps {
     fagsak: IFagsak;
 }
 
 const FastsettVedtak: React.FunctionComponent<IProps> = ({ fagsak }) => {
-    const [state, dispatch] = useFastsettReducer(fagsak);
+    const history = useHistory();
+    const context = useFastsettVedtakContext();
+    const dispatch = useFastsettVedtakDispatch();
     const [visFeilmeldinger, settVisFeilmeldinger] = React.useState(false);
+    const [opprettelseFeilmelding, settOpprettelseFeilmelding] = React.useState('');
+
+    const aktivBehandling = fagsak.behandlinger.find(
+        (behandling: IBehandling) => behandling.aktiv === true
+    );
+
+    if (!aktivBehandling) {
+        return (
+            <div>
+                <Normaltekst>Ingen aktiv behandling</Normaltekst>
```

Comment:
To my understanding, it is an internal error if there is no active behandling. Why dont we use the same way to display all the error messages through SkjemaGruppe.feil?

Reply:
Because it is not possible for a case worker to work on a case without an active case.